src/components/TextInputFocusable/index.js

```diff
@@ -252,13 +236,11 @@ class TextInputFocusable extends React.Component {
     handlePastedHTML(html) {
         const parser = new ExpensiMark();
         const markdownText = parser.htmlToMarkdown(html);
-        const beforeCursorText = this.textInput.value.substring(0, this.selection.start);
-        const afterCursorText = this.textInput.value.substring(this.selection.end);
-        this.textInput.value = beforeCursorText + markdownText + afterCursorText;
-        const newCursorPosition = beforeCursorText.length + markdownText.length;
-        this.setState({selection: {start: newCursorPosition, end: newCursorPosition}});
-        this.updateNumberOfLines();
-        this.props.onChangeText(this.textInput.value);
+        try {
+            document.execCommand('insertText', false, markdownText);
+            this.updateNumberOfLines();
+        // eslint-disable-next-line no-empty
+        } catch (e) {}
```

Comment:
The `execCommand` API is deprecated. Is there an alternative to this?

Also, could you add a comment to explain why do we need an empty `catch {}` block here?

Reply:
@HorusGoul Its deprecated but I didn't find any direct alternative. If we don't use it then we have to implement our own framework to manage undo redo and other operations. There are few WYSIWYG editors out there which does this but we are not currently using it. 

After a careful inspection I come to the decision of using it, previously I was doing the manual paste but it lacked Undo functionality..

This is still supported for almost all browsers https://caniuse.com/?search=execCommand.
Also, I checked RN-WEB to see how they manage Copy to clipboard. They are also using it https://github.com/necolas/react-native-web/blob/98dcefd9041c10cc95a2385202498b274d926a6b/packages/react-native-web/src/exports/Clipboard/index.js#L50

I just borrowed the catch block from RN-web. 

We can think about of changing to different alternative when we redesign our composer  but I would say ATM its the best bet.

Reply:
Great! Let's merge then :)